src/components/common/header/index.astro

```diff
@@ -4,10 +4,55 @@
 import Container from "@/layouts/container/index.astro"
 
 ---
-<header class="bg-black h-12">
+<header class="bg-black h-20 fixed z-50 w-full items-center">
     <Container>
-        <section class="flex items-center h-full">
-            <p class="text-white">Header</p>
+        <section class="flex items-center justify-between cursor-pointer">
+            <div class="flex">
+                <div class="h-18 w-40 py-4">
+                    <img src="./public/logo/Lightfield-logo.png" alt="">
```

Comment:
@Nandiniverma2024 add a class with properties h-full and w-full to this image tag. Import path should be /logo/.. and not ./public.